.github/main.swift

```diff
@@ -213,8 +213,16 @@ class ReadmeGenerator {
     func generateReadme() {
         print("Start")
         do {
-            let applicationsData = try Data(contentsOf: URL(fileURLWithPath: FilePaths.applications.rawValue))
-            let categoriesData = try Data(contentsOf: URL(fileURLWithPath: FilePaths.categories.rawValue))
+            // get current file path:
+            let thisFilePath:String = #file
+            var url = URL(fileURLWithPath: thisFilePath)
+
+            //cd ../ to the root folder: (delete `.github/main.swift`
```

Comment:
oh, if you talking about this comment- no worry. it's more tricky. it goes always from the locations, where the file is stored. it's independent from the path, where you start it from! 🆗